development/includes/_top-nav.pug

```diff
@@ -1,13 +1,13 @@
-// Classes are currently placeholder with zero styling. Removed bootstrap
+script(src="/resources/js/_top-nav.js")
```

Comment:
Sis wtf? I literally have you the correct code to use. This is still wrong and you're worry to much about functionality. Please follow directions, that's perhaps the biggest skill you can learn right now. The coding part will be easy.